packages/flutter/lib/src/foundation/tree_diagnostics_mixin.dart

```diff
@@ -6,6 +6,16 @@ import 'package:meta/meta.dart';
 
 import 'print.dart';
 
+/// Returns a 5 character long hexadecimal string generated from
+/// Object.hashCode's 20 least-significant bits.
+String shortHash(Object object) {
+  return object.hashCode.toUnsigned(20).toRadixString(16).padLeft(5, '0');
+}
+
+/// Returns a summary of the runtime type and hash code of `object`.
+String describeIdentity(Object object) =>
```

Comment:
style guide says not to split lines that use =>, so this should either be one line or have a block and an explicit return